bee-network/examples/common/utf8msg.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt;
+
+#[derive(Clone)]
+pub struct Utf8Message {
```

Comment:
What's the purpose of this type? It seems to fairly transparently wrap `String::from_utf8` and `str::as_bytes`.